cmd/cloud/deployment.go

```diff
@@ -3,6 +3,10 @@ package cloud
 import (
 	"io"
 
+	"github.com/astronomer/astro-cli/astro-client"
+
+	"github.com/astronomer/astro-cli/cloud/deployment/workerqueue"
+
```

Comment:
```suggestion
	"github.com/astronomer/astro-cli/astro-client"

	"github.com/astronomer/astro-cli/cloud/deployment/workerqueue"

```

nitpick: for readability we can club imports paths, for example:
```
import (
    <stdlib>

    <third_party_lib>

    <internal_imports>
)
```

```
import (
	"io"

	"github.com/pkg/errors"
	"github.com/spf13/cobra"

	"github.com/astronomer/astro-cli/astro-client"
	"github.com/astronomer/astro-cli/cloud/deployment/workerqueue"
        airflowversions "github.com/astronomer/astro-cli/airflow_versions"
	"github.com/astronomer/astro-cli/cloud/deployment"
	"github.com/astronomer/astro-cli/pkg/httputil"
)
```

Reply:
I am surprised